runtime/src/accounts_db.rs

```diff
@@ -764,22 +808,40 @@ impl AccountsDB {
         }
 
         self.handle_reclaims(&reclaims);
+        reclaims_time.stop();
+        debug!(
+            "clean_accounts: {} {} {} {}",
+            accounts_scan, store_counts_time, purge_filter, reclaims_time
+        );
     }
 
     fn handle_reclaims(&self, reclaims: &[(Slot, AccountInfo)]) {
         let mut dead_accounts = Measure::start("reclaims::remove_dead_accounts");
-        let mut dead_slots = self.remove_dead_accounts(reclaims);
+        let dead_slots = self.remove_dead_accounts(reclaims);
         dead_accounts.stop();
+        self.dead_slots.write().unwrap().extend(dead_slots);
+    }
+
+    pub fn handle_dead_slots(&self) {
+        let mut dead_slots_w = self.dead_slots.write().unwrap();
+        let dead_slots = dead_slots_w.clone();
```

Comment:
How about using [`std::mem::replace`](https://doc.rust-lang.org/std/mem/fn.replace.html) like [this](https://doc.rust-lang.org/std/mem/fn.replace.html)? We can avoid that artificial `.clone()`.